pkg/istio/rbac/common/util.go

```diff
@@ -246,25 +287,31 @@ func ParseAssertionResource(domainName zms.DomainName, assertion *zms.Assertion)
 
 // CheckAthenzSystemDisabled checks if athenz domain is systematically disabled, if so, controller skips processing current
 // role member
-func CheckAthenzSystemDisabled(roleMember *zms.RoleMember) (bool, error) {
-	if roleMember == nil {
-		return false, fmt.Errorf("got an empty role Member: %s, skipping", roleMember.MemberName)
+func CheckAthenzSystemDisabled(member interface{}) (bool, error) {
+	if member == nil {
+		return false, fmt.Errorf("got an empty role Member: %s, skipping", GetMemberName(member))
 	}
-	if roleMember.SystemDisabled != nil && *roleMember.SystemDisabled != 0 {
-		return false, fmt.Errorf("member %s is system disabled", roleMember.MemberName)
+
+	systemDisabled := getMemberSystemDisabled(member)
+
+	if systemDisabled != nil && *systemDisabled != 0 {
+		return false, fmt.Errorf("member %s is system disabled", GetMemberName(member))
 	}
 	return true, nil
 }
 
 // CheckAthenzMemberExpiry checks if Expiration field (timezone UTC) is set in the roleMember object, and then
 // checks if expiration date surpasses current time
-func CheckAthenzMemberExpiry(roleMember *zms.RoleMember) (bool, error) {
-	if roleMember == nil {
-		return false, fmt.Errorf("got an empty role Member: %s, skipping", roleMember.MemberName)
+func CheckAthenzMemberExpiry(member interface{}) (bool, error) {
+	if member == nil {
+		return false, fmt.Errorf("got an empty role Member: %s, skipping", GetMemberName(member))
```

Comment:
Same here as well, won't this just be an empty string?

Reply:
True, will change